docs/_includes/customizer-variables.html

```diff
@@ -33,28 +33,28 @@ <h2 id="colors">Colors</h2>
     <input id="input-@brand-navy" type="text" value="#37517e" data-var="@brand-navy" class="form-control"/>
   </div>
 </div>
-<h3 id="blues-(deprecated)">Blues (Deprecated)</h3>
+<h3 id="blues">Blues</h3>
 <div class="row">
   <div class="col-xs-4">
     <label for="input-@brand-blue-minus-20">@brand-blue-minus-20</label>
-    <input id="input-@brand-blue-minus-20" type="text" value="@brand-navy-minus-20" data-var="@brand-blue-minus-20" class="form-control"/>
+    <input id="input-@brand-blue-minus-20" type="text" value="#111824" data-var="@brand-blue-minus-20" class="form-control"/>
```

Comment:
That looks weird. I see the same pattern in this file a lot. Also why are we changing the variable to a hardcoded color only for some?

Reply:
Oh, I see, this is an auto generated file, it just needs a unique name, so this is fine.